saleor/order/actions.py

```diff
@@ -630,9 +630,8 @@ def _move_order_lines_to_target_fulfillment(
         try:
             deallocate_stock(lines_to_dellocate)
         except AllocationError as e:
-            logger.warning(
-                f"Unable to deallocate stock for line {', '.join(e.order_lines)}."
-            )
+            lines = [str(line) for line in e.order_lines]
+            logger.warning(f"Unable to deallocate stock for line {', '.join(lines)}.")
```

Comment:
Maybe we should add order_line.pk to logs instead of `str(order_line)`.
```
    def __str__(self):
        return (
            f"{self.product_name} ({self.variant_name})"
            if self.variant_name
            else self.product_name
        )
  ```

Reply:
IMO we should user logger.warning("warning-msg", extra={"your-error": "details}). with this way we don't include any important details as a string directly to the record.